Buform.Example.Core/DigitGenerationItem.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Linq.Expressions;
+using System.Windows.Input;
+using MvvmCross.Commands;
+
+namespace Buform.Example.Core
+{
+    public sealed class DigitGenerationItem : FormItem<int>
+    {
+        private const int MinValue = 0;
+        private const int MaxValue = 1000;
+
+        private readonly Random _random;
+
+        public string? Label { get; set; }
+        public ICommand RegenerateCommand { get; }
```

Comment:
Are we regenerating something here? I think it should be just `GenerateCommand`

Reply:
The command callback function should be named by the command name as well to be clear. E.g. Regenerate -> Generate